crates/risc0/src/guest.rs

```diff
@@ -34,13 +35,13 @@ impl ZkvmGuest for Risc0Guest {
     }
 
     fn verify_with_assumptions(journal: &[u8], code_commitment: &Self::CodeCommitment) {
-        env::verify(code_commitment.0, journal)
+        env::verify(*code_commitment, journal)
             .expect("Assumption API verify error should be infallible")
     }
 }
 
 impl Zkvm for Risc0Guest {
-    type CodeCommitment = Risc0MethodId;
```

Comment:
Well, `type Risc0MethodId = Digest;` is actually means something. Although it's not required at all.